components/brave_wallet/browser/simulation_service.cc

```diff
@@ -150,9 +155,7 @@ GURL SimulationService::GetScanTransactionURL(const std::string& chain_id,
                                               const std::string& language) {
   std::string spec = base::StringPrintf(
       "%s/%s/%s", kBlowfishBaseAPIURL,
-      GetRelativeScanPath(chain_id, coin).c_str(),
-      coin == mojom::CoinType::SOL ? "transactions" : "transaction");
-
+      GetRelativeScanPath(chain_id, coin).c_str(), "transactions");
```

Comment:
nit: can we have a DCHECK on supported coin arg?

Reply:
Added: https://github.com/brave/brave-core/pull/21609/commits/99e1648dd202ccacd59eae2b593ff448be84a68d